hs_geographic_feature_resource/models.py

```diff
@@ -1,15 +1,20 @@
+from lxml import etree
+
 from django.db import models
 from django.contrib.contenttypes.fields import GenericRelation
 
 from mezzanine.pages.page_processors import processor_for
 
-from hs_core.models import BaseResource, ResourceManager, resource_processor,\
-                           CoreMetaData, AbstractMetaDataElement
+from dominate.tags import legend, table, tbody, tr, td, th, h4, div
 
-from lxml import etree
+from hs_core.models import BaseResource, ResourceManager, resource_processor, \
+    CoreMetaData, AbstractMetaDataElement
 
+from hs_core.hydroshare.utils import add_metadata_element_to_xml
 
-# Create your models here.
+
+# TODO: Pabitra - this model is not being used anymore. however, deleting this class
+# causes the migration to hang during the build process
```

Comment:
@hyi After deleting the model class I ran ```makemigrations``` command which created the needed new migration file. But running the ```migrate``` command then hangs in Jenkins. In local development the ```migrate``` command prompts to delete the content type (yes/no). I select yes for this prompt. I am not sure if we have to change the hsctl scripts to handle this case. @mjstealey or @aphelionz any suggestion to resolve this?